io/plugins/eu.esdihumboldt.hale.io.gml.ui/src/eu/esdihumboldt/hale/io/gml/ui/GmlWriterSettingsPage.java

```diff
@@ -99,9 +112,44 @@ protected void createContent(Composite page) {
 		// default
 		nilReason.setSelection(true);
 
+		Group writeFormat = new Group(page, SWT.NONE);
+		writeFormat.setLayout(GridLayoutFactory.swtDefaults().numColumns(2).create());
+		writeFormat.setText("Formatted number output for geometry");
+		groupData.applyTo(writeFormat);
+
+		Label lbl = new Label(writeFormat, SWT.NONE);
+		lbl.setLayoutData(GridDataFactory.swtDefaults().align(SWT.END, SWT.CENTER).create());
+		lbl.setText("Format: ");
+
+		formattedText = new Text(writeFormat, SWT.SINGLE | SWT.BORDER);
+		formattedText.setLayoutData(GridDataFactory.swtDefaults().align(SWT.FILL, SWT.CENTER)
+				.grab(true, false).create());
+		// filler
+		Label lblfiller = new Label(writeFormat, SWT.NONE);
+		lblfiller.setLayoutData(GridDataFactory.swtDefaults().align(SWT.END, SWT.CENTER).create());
+
+		Label formatDesc = new Label(writeFormat, SWT.NONE);
+		formatDesc.setLayoutData(GridDataFactory.swtDefaults().align(SWT.FILL, SWT.CENTER)
+				.grab(true, false).create());
+		formatDesc.setText("(for example 00000.000)");
```

Comment:
I think having leading zeros is a case that would not be used often (if at all), so I would prefer an example / default that does not include them.